content/md/productivity/manage-your-rules.md

```diff
@@ -246,6 +231,22 @@ You can also execute several rules at the same time:
 
 ![image](../img/Rules_BulkExecution.png)
 
+# Duplicate a rule
+
+To duplicate a rule, just access the edit form of the rule you want to duplicate, then click on `...` and `Save and execute`. The creation popin opens and you can choose a code and a label for your new rule.
+
+# Define a status to a rule
+
+Sometimes, you may need to create/update a rule, save it, but ***not*** execute it automatically, because you have to refine it for instance. To do so, you can use the `Status` property.
```

Comment:
```suggestion
Sometimes, you may need to create/update a rule, save it, but ***not*** execute it automatically, because you will need to finetune it later for instance. To do so, you can use the `Status` property.
```